docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,53 +1,54 @@
 [[connect-to-elasticsearch]]
 == Add your data
 ++++
-<titleabbrev>Add data</titleabbrev>
+<titleabbrev>Add an integration</titleabbrev>
 ++++
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data is to use one of our many integrations.
+Alternatively, you can add a sample data set or upload a file.
 
-[role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
 [float]
 [[add-data-tutorial-kibana]]
-=== Add data
+=== Add an integration
 
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
+Elastic integrations are a streamlined way to connect your data to the Elastic Stack.
+Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB,
+as well as many generic input types like log files.
+Each integration comes pre-packaged with assets that support your needs:
```

Comment:
We are now referring to pretty much every ingest option here as an integration. This feels specific to agent / beats. What are your thoughts on including enterprise search and others into the mix? For other types of integrations outside of agent / beats, the bullet points no longer fit. Should we provide less concrete details and more of a high level overview of what's available?